### ISSUE TITLE
Android SDK documentation does not mention that com.google.android.gms:play-services-ads-identifier is required

### ISSUE THREAD

**Issue by aeirola** [PiwikPRO/PPMS-PublicDocs#520]
Without explicitly depending on the library, the app crashes with the following stack trace:

```
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient;
at pro.piwik.sdk.Tracker$1.doInBackground(Tracker.java:225)
at pro.piwik.sdk.Tracker$1.doInBackground(Tracker.java:220)
at android.os.AsyncTask$3.call(AsyncTask.java:394)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
```

thus any user of the SDK needs to include 

```
implementation "com.google.android.gms:play-services-ads-identifier:17.0.0"
```

in their `build.gradle` configuration, or depend on some other library which includes the library.

Alternatively, the Android SDK itself could specify the required library as a dependency.

**Comment by aeirola** [comment]
Apparently this only happens if you have a dependency which increases the version of `com.google.android.gms:play-services-basement` from 11.0.4 to 15.0.0 or higher. This is why it probably works on demo applications.

**Comment by harmasz** [comment]
Sorry for the long wait.
We can confirm that we will be working on this in the library in the next couple of weeks.

**Comment by MGaetan89** [comment]
It looks like this was addressed in [1.1.6](https://github.com/PiwikPRO/sdk-framework-android/releases/tag/1.1.6).
So I guess that this can be closed now.

**Comment by glothriel** [comment]
Closing then, feel free to reopen if the problem re-occurs.
